packages/@react-spectrum/form/test/Form.test.js

```diff
@@ -183,5 +197,45 @@ describe('Form', function () {
       let form = getByRole('form');
       expect(form.elements['picker'].value).toEqual('one');
     });
+
+    it('contextual help should not be disabled nor should its dismiss button be disabled', () => {
+      matchMedia.useMediaQuery('(max-width: 700px)');
+
+      let {getByRole, getByLabelText} = render(
+        <Provider theme={theme}>
+          <Form aria-label="Test" isDisabled>
+            <Picker
+              name="picker"
+              defaultSelectedKey="one"
+              label="Test Picker"
+              contextualHelp={(
+                <ContextualHelp>
+                  <Header>What is it good for?</Header>
+                  <Content>Absolutely nothing.</Content>
+                </ContextualHelp>
+              )}>
+              <Item key="one">One</Item>
+              <Item key="two">Two</Item>
+              <Item key="three">Three</Item>
+            </Picker>
+          </Form>
+        </Provider>
+      );
+
+      let button = getByLabelText('Help');
+      triggerPress(button);
+
+      let dialog = getByRole('dialog');
+      userEvent.tab();
+
+      let dismissButton = within(dialog).getByRole('button');
+      expect(document.activeElement).toBe(dismissButton);
+
+      triggerPress(dismissButton);
+      act(() => {jest.runAllTimers();});
+      act(() => {jest.runAllTimers();});
```

Comment:
Why do you run them twice?

Reply:
focus movement, check in the dialog trigger tests, you'll see the same thing as well as an explanation of why